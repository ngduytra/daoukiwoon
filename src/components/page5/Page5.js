import React from 'react';
import './page5.styles.scss'
import Computer from '../../assets/img/Computer.png'

const Page5 = () => {
    return (
        <div id='index-5'>
            <div className="content">
                <div className='left'>
                    <div className='title'>SALES</div>
                    <div className='subtitle'>Create Homepage Youtube & Instagram</div>
                    <div className='button'>Fieldmake</div>
                </div>
                <img src={Computer} />
            </div>
        </div>
    )
}

export default Page5;