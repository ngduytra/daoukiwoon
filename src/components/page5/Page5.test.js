import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';
import React from 'react';
import Page5 from './Page5';

configure({adapter: new Adapter()});

it('expect to render Page5 component', ()=>{
    expect(shallow(<Page5/>)).toMatchSnapshot();
})