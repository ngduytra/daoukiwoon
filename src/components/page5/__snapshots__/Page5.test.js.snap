// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Page5 component 1`] = `ShallowWrapper {}`;
